Packages/com.unity.inputsystem/Tests/IntegrationTests/IntegrationTests.cs

```diff
@@ -0,0 +1,24 @@
+using NUnit.Framework;
+using UnityEngine.InputSystem;
+using UnityEngine.InputSystem.LowLevel;
+
```

Comment:
maybe it makes sense to leave a comment here (or maybe better a readme.txt file in this folder?) pointing out to critical eyes that there are many more tests outside of the package?

Reply:
Doh shit, that's what I planned to do but looks like I just plain forgot. Thanks for catching it. Adding.

Reply:
Fixed